lib/streamlit/elements/metric.py

```diff
@@ -102,12 +103,14 @@ def metric(
 
         help : str
             An optional tooltip that gets displayed next to the metric label.
+            Streamlit only displays the tooltip when
+            ``label_visibility="visible"``.
 
         label_visibility : "visible", "hidden", or "collapsed"
-            The visibility of the label. If "hidden", the label doesn't show but there
-            is still empty space for it (equivalent to label="").
-            If "collapsed", both the label and the space are removed. Default is
-            "visible".
+            The visibility of the label. The default is ``"visible"``. If this
+            is ``"hidden"``, Streamlit displays an empty spacer instead of the
```

Comment:
TIL that `spacer` is a word 😅

Reply:
I could say "empty space" or "empty block" if that sounds better, but "spacer" seemed like a clear and casual descriptor. 😀